From 666ab563815e6f62c593279bd8f5dc0e9d2ad976 Mon Sep 17 00:00:00 2001
From: Mike Hommey <mh@glandium.org>
Date: Mon, 22 Mar 2010 11:45:42 +0100
Subject: [PATCH] Don't error-out when run-time libsqlite is older than
 build-time libsqlite

In Debian, we have shlibs to handle this.

---
 storage/mozStorageService.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/storage/mozStorageService.cpp b/storage/mozStorageService.cpp
index 5b3e6ed1ae..aafd5aa5b9 100644
--- a/storage/mozStorageService.cpp
+++ b/storage/mozStorageService.cpp
@@ -184,6 +184,7 @@ already_AddRefed<Service> Service::getSingleton() {
     return do_AddRef(gService);
   }
 
+#if 0
   // Ensure that we are using the same version of SQLite that we compiled with
   // or newer.  Our configure check ensures we are using a new enough version
   // at compile time.
@@ -208,6 +209,7 @@ already_AddRefed<Service> Service::getSingleton() {
     }
     MOZ_CRASH("SQLite Version Error");
   }
+#endif
 
   // The first reference to the storage service must be obtained on the
   // main thread.
